### PR TITLE
fix: Layout for sortable column headers

### PR DESCRIPTION
**Pull request by JCarmine**

### CODE REVIEW

File: src/components/Table/Table.module.scss [hunk 1]
```diff
@@ -15,6 +15,7 @@
   }
 
   .table {
+    margin-top: 1px;
```
Comment (nathanyoung) [inline src/components/Table/Table.module.scss:18]: @JCarmine Why is this necessary?

Comment (JCarmine) [reply src/components/Table/Table.module.scss:18]: To make the focus ring rendered by Chrome fully visible; the ring's width is 2px, and it was partially obscured. We can chat about it in the am, and decide whether we care or not.
